### ISSUE TITLE
gen_uint() method cant be called on ThreadRng due to unsatisfied Trait bounds

### ISSUE THREAD

**Issue by PrisionMike** [Aatch/ramp#124]
The doc code doesn't work. I tried different rngs from rand. all of these show the same error at build
```
let gma: Int = rng.gen_uint(256);
   |                        ^^^^^^^^ method cannot be called on `ThreadRng` due to unsatisfied trait bounds
.cargo\registry\src\github.com-1ecc6299db9ec823\rand-0.8.3\src\rngs\thread.rs:61:1
   |
61 | pub struct ThreadRng {
   | --------------------
   | |
   | doesn't satisfy `ThreadRng: RandomInt`
   | doesn't satisfy `ThreadRng: rand::Rng`
   |
   = note: the following trait bounds were not satisfied:
           `ThreadRng: rand::Rng`
           which is required by `ThreadRng: RandomInt`
```

**Comment by PrisionMike** [comment]
rustc 1.54.0-nightly
ramp = 0.5.9
rand = 0.8.3

**Comment by PrisionMike** [comment]
I used rand = 0.5 and it runs now. The issue may be closed as the dependencies do say rand = 0.5

**Comment by rozbb** [comment]
Version 0.6 now uses `rand = 0.8` as a dep, so that should fix the issue. Sorry about that.
